client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,27 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
+
 
 module.exports = class PasswordView extends StepView
+
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click button': 'doSubmit'
+
+
+    getDataFromDOM: ->
+        # TODO: get user info
+        # from server
+        return {
+            email: 'toto@cozycloud.cc'
+            public_name: 'toto'
+            timezone: timezones[0]
+            password: @$('input[name=password]').val()
+            allow_stats: false
+        }
```

Comment:
The password component should send only the password from its form/input (not prefilled). So only two properties should be in the request body here: ```password``` and ```onboardedSteps``` (cf doc [here](https://github.com/cozy/cozy-proxy/blob/development/client/doc/onboardingDataHandling.md#views-data-behaviour-client-side)) like:
```coffee
{ 
    password: @$('input[name=password]').val(),
    onboardedSteps: ['welcome', 'agreement', 'password']
}
```

Other data are handled by other steps separately from this component.